ext/webserial/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+[package]
+name = "deno_webserial"
+version = "0.1.0"
+authors = ["the Deno authors"]
+edition = "2018"
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/denoland/deno"
+description = "Implementation of Web Serial API for Deno"
+
+[lib]
+path = "lib.rs"
+
+[dependencies]
+deno_core = { version = "0.101.0", path = "../../core" }
+serde = { version = "1.0.129", features = ["derive"] }
+# needed for a type
+serialport = "4.0.1"
+tokio = { version = "1.10.1", features = ["full"] }
+tokio-serial = "5.4.1"
```

Comment:
The dependency on libudev-sys is somewhat problematic (causes CI to fail because pkg-config can't find libudev, indicative of what users are going to run into) but it's optional and can probably be worked around by forking libudev-sys.

tokio-serial according to its README isn't maintained anymore.

Reply:
> tokio-serial according to its README isn't maintained anymore.

the README is a bit inaccurate: even though it had that warning for a while, it got still some new releases, also following the issue linked in the README shows that it has been handed over to a new maintainer a few months back